hcaptcha/api/hcaptcha.api

```diff
@@ -0,0 +1,103 @@
+public final class com/stripe/hcaptcha/BuildConfig {
+	public static final field BUILD_TYPE Ljava/lang/String;
+	public static final field DEBUG Z
+	public static final field LIBRARY_PACKAGE_NAME Ljava/lang/String;
+	public fun <init> ()V
+}
+
+public final class com/stripe/hcaptcha/HCaptcha$Companion {
+	public final fun getClient (Landroidx/fragment/app/FragmentActivity;)Lcom/stripe/hcaptcha/HCaptcha;
+	public final fun getClient (Landroidx/fragment/app/FragmentActivity;Lcom/stripe/hcaptcha/config/HCaptchaInternalConfig;)Lcom/stripe/hcaptcha/HCaptcha;
+}
+
+public final class com/stripe/hcaptcha/HCaptchaDialogFragment$Companion {
+	public final fun newInstance (Lcom/stripe/hcaptcha/config/HCaptchaConfig;Lcom/stripe/hcaptcha/config/HCaptchaInternalConfig;Lcom/stripe/hcaptcha/HCaptchaStateListener;)Lcom/stripe/hcaptcha/HCaptchaDialogFragment;
+	public final fun test ()Lcom/stripe/hcaptcha/HCaptchaDialogFragment;
+}
+
+public final class com/stripe/hcaptcha/HCaptchaError$Companion {
+	public final fun fromId (I)Lcom/stripe/hcaptcha/HCaptchaError;
+}
+
+public final class com/stripe/hcaptcha/HCaptchaHtmlKt {
+}
+
+public final class com/stripe/hcaptcha/HCaptchaStateListener$Creator : android/os/Parcelable$Creator {
```

Comment:
Anything suffixed with `$Creator` or `$$serializer` are expected, but I think we can get rid of the others.

Reply:
I added the same `@file:RestrictTo` here, but these are still showing up. I'm wondering if there's another way to eliminate these.

Reply:
specifically the enum companions are showing up regardless of the inclusion of the `@file:RestrictTo` line.

Reply:
Those are expected. I think the apiDump looks good now!